apps/calidum-rotae/base/calidum-rotae-service/deployment.yaml

```diff
@@ -23,16 +23,28 @@ spec:
     spec:
       containers:
         - name: calidum-rotae-service
-          image: ghcr.io/clubcedille/calidum-rotae-service:latest
+          image: ghcr.io/clubcedille/calidum-rotae-service:1.0.3
           imagePullPolicy: Always
           env:
             - name: CALIDUM_ROTAE_SERVICE_API_KEY
               valueFrom:
                 secretKeyRef:
                   name: calidum-rotae-secrets
                   key: CALIDUM_ROTAE_SERVICE_API_KEY
+            - name: OTEL_SERVICE_NAME
+              value: "calidum"
+            - name: OTEL_EXPORTER_OTLP_TRACES_ENDPOINT
+              value: "http://coroot.coroot.svc.cluster.local:8080/v1/traces"
+            - name: OTEL_EXPORTER_OTLP_LOGS_ENDPOINT
+              value: "http://coroot.coroot.svc.cluster.local:8080/v1/logs"
+            - name: OTEL_EXPORTER_OTLP_PROTOCOL
+              value: "http/protobuf"
+            - name: OTEL_METRICS_EXPORTER
+              value: "none"
+            - name: OTEL_EXPORTER_OTLP_HEADERS
+              value: "x-api-key="
           ports:
             - containerPort: 3000
           command:
             # ['/bin/sh', '-c', 'source /vault/secrets/calidum-rotae-service-env && /calidum_rotae_service --port 3000 --discord_provider_hostname discord-provider-service --discord_provider_port 5000 --email_provider_hostname email-provider-service --email_provider_port 4000 --otel_otlp_exporter_host opentelemetry-collector-collector-headless.opentelemetry-collector.svc.cluster.local  --otel_otlp_exporter_port 4318 --allowed_domains "https://*.cedille.club,https://*.etsmtl.ca"']
-            ['/bin/sh', '-c', '/calidum_rotae_service --port 3000 --discord_provider_hostname discord-provider-service --discord_provider_port 5000 --email_provider_hostname email-provider-service --email_provider_port 4000 --otel_otlp_exporter_host opentelemetry-collector-collector-headless.opentelemetry-collector.svc.cluster.local  --otel_otlp_exporter_port 4318 --allowed_domains "http://locahost,http://localhost:3000,https://*.cedille.club,https://*.etsmtl.ca,https://*.prod.cedille.club"']
+            ['/bin/sh', '-c', '/calidum_rotae_service --port 3000 --discord_provider_hostname discord-provider-service --discord_provider_port 5000 --email_provider_hostname email-provider-service --email_provider_port 4000 --otel_otlp_exporter_host coroot.coroot.svc.cluster.local  --otel_otlp_exporter_port 8080 --allowed_domains "http://locahost,http://localhost:3000,https://*.cedille.club,https://*.etsmtl.ca,https://*.prod.cedille.club"']
```

Comment:
we can keep https://*.cedille.club and remove *.prod.cedille.club in allowed_domains since the first one includes the second one